Wikipedia/UI-V5/WMFSaveableTitleCollectionViewCell+Subclass.h

```diff
@@ -0,0 +1,48 @@
+//
+//  WMFSaveableTitleCollectionViewCell+Subclass.h
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 9/2/15.
+//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#import "WMFSaveableTitleCollectionViewCell.h"
+
+@interface WMFSaveableTitleCollectionViewCell ()
```

Comment:
Should have a Category name here like `(Subclasses)`

In general class extensions are for private information to the class itself, rather than for public consumption via a separate header. The canonical example is UIGestureRecognizerSubclass.h.